Errors Related to `HttpStatusException` When Running Unit Tests

I think I've had this for a while now:

In serial execution, so it's clear which tests caused the errors:
```
[INFO] Running com.redhat.rhjmc.containerjfr.net.web.handlers.ReportGetHandlerTest
io.vertx.ext.web.handler.impl.HttpStatusException: Not Found

[INFO] Running com.redhat.rhjmc.containerjfr.net.web.handlers.TargetRecordingGetHandlerTest
io.vertx.ext.web.handler.impl.HttpStatusException: Internal Server Error
Caused by: java.lang.NullPointerException
io.vertx.ext.web.handler.impl.HttpStatusException: Not Found
```

The errors don't cause the test run to fail, so I've never really thought much about them, but looking more closely I guess it probably does mean something is wrong. Am I missing a dependency maybe?

--- a/src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/ReportGetHandlerTest.java
+++ b/src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/ReportGetHandlerTest.java
@@ -114,7 +114,6 @@ class ReportGetHandlerTest {
 
         HttpStatusException ex =
                 Assertions.assertThrows(HttpStatusException.class, () -> handler.handle(ctx));
-        ex.printStackTrace();
         MatcherAssert.assertThat(ex.getStatusCode(), Matchers.equalTo(404));
     }
 }

--- a/src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/TargetRecordingGetHandlerTest.java
+++ b/src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/TargetRecordingGetHandlerTest.java
@@ -223,7 +223,6 @@ class TargetRecordingGetHandlerTest {
 
         HttpStatusException ex =
                 Assertions.assertThrows(HttpStatusException.class, () -> handler.handle(ctx));
-        ex.printStackTrace();
         MatcherAssert.assertThat(ex.getStatusCode(), Matchers.equalTo(404));
     }
 
@@ -246,7 +245,6 @@ class TargetRecordingGetHandlerTest {
 
         HttpStatusException ex =
                 Assertions.assertThrows(HttpStatusException.class, () -> handler.handle(ctx));
-        ex.printStackTrace();
         MatcherAssert.assertThat(ex.getStatusCode(), Matchers.equalTo(500));
     }
 }

